.github/workflows/f5-cla.yml

```diff
@@ -0,0 +1,52 @@
+name: F5 CLA
+
+on:
+  issue_comment:
+    types:
+      - created
```

Comment:
This means they have to sign when opening an issue?

Reply:
This is to read the comment when someone agrees to the CLA. GitHub doesn't differentiate issues and PRs, but I'm not sure why they only use "created" in the Action docs.